python/redis_rpc/__init__.py

```diff
@@ -45,22 +45,31 @@ def warn_if_no_socket_timeout(redis):
                     'problems between Redis and RPC client or server.')
 
 
-def escape_for_logs(v):
+def format_for_logs(s, *, verbose):
+    if isinstance(s, bytes):
+        s = s.decode()
+    if not verbose and len(s) > 120:
+        return s[:120] + '...'
     # quick and dirty way to avoid log forging/injection
-    return json.dumps(v)
+    return json.dumps(s)
 
 
-def log_request(func_name, req_bytes, exception, msg):
+def log_request(func_name, req_bytes, exception, result, msg, *, verbose):
+    parts = [
+        func_name,
+        format_for_logs(req_bytes, verbose=verbose),
+        format_for_logs('%s: %s' % (type(exception).__name__,
+                                    str(exception)), verbose=verbose)
+        if exception else '-',
```

Comment:
Tiny, aesthetics: splitting the `X if Y else Z` construct into multiple lines this way makes it noticeably less readable for me.  Can we change this?  Maybe make this `format_exception_for_logs(exception, verbose=verbose)`, push the `if` into it?

Reply:
Updated, how about now?